ssr-testing/pages/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import Head from 'next/head';
+import Link from 'next/link';
+
+export default function Home() {
+  return (
+    <>
+      <Head>
+        <title>Home</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <h1>Components</h1>
+        <ul>
+          <li>
+            <Link href="./radio-group">RadioGroup</Link>
```

Comment:
Getting this error 

![image](https://user-images.githubusercontent.com/175330/107936174-3c1a9380-6f7a-11eb-9896-d14920d4e900.png)

Looks like we don't need the dot `.`?